pkg/generated/bindata.go

```diff
@@ -290,7 +290,7 @@ metadata:
     csi.openshift.io/managed: "true"
 spec:
   attachRequired: true
-  podInfoOnMount: true
+  podInfoOnMount: false
 `)
```

Comment:
Did we intend to change this? I don't see corresponding change in YAML.

Reply:
I guess whoever updated `csidriver.yaml` last time didn't run `make update`.